### PR TITLE
Banner design changes

### PR DESCRIPTION
**Pull request by alycrys**

### CODE REVIEW

**Comment by libertymayc** [comment]
lgtm, changeset? :)

**Review by Fercas123 — APPROVED** [review]
LGTM

**Review by libertymayc — DISMISSED** [review]

File: .changeset/serious-parrots-speak.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/lab": patch
+---
+
+Banner style changes to use spacing foundation
```
Comment (joshwooding) [inline .changeset/serious-parrots-speak.md:5]:
```suggestion
Updated Banner to use the new spacing foundation
```

**Review by joshwooding — CHANGES_REQUESTED** [review]

File: packages/lab/src/banner/Banner.tsx [hunk 2]
```diff
@@ -68,10 +68,7 @@ export const Banner = forwardRef<HTMLDivElement, BannerProps>(function Banner(
       {...rest}
       aria-live="polite"
     >
-      <StatusIndicator
-        className={clsx(withBaseName("icon"), className)}
-        status={status}
-      />
+      <StatusIndicator status={status} className={clsx(withBaseName("icon"))} />
```
Comment (joshwooding) [inline packages/lab/src/banner/Banner.tsx:71]:
```suggestion
      <StatusIndicator status={status} className={withBaseName("icon")} />
```

File: packages/lab/stories/banner/banner.stories.tsx [hunk 3]
```diff
@@ -76,9 +76,12 @@ export const MultipleLines = (props: BannerProps) => {
         <Banner {...props}>
           <BannerContent>
             <div>
-              Our guidance for hyphen and dash usage differs from that of the
-              “AP Stylebook” and is aligned with the “J.P. Morgan Brand
-              Guidelines” (also known as the Masterbrand guide).
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
```
Comment (joshwooding) [inline packages/lab/stories/banner/banner.stories.tsx:79]: We should be trying to avoid using lorem ipsum. Is there a better example text we can use here?

**Review by joshwooding — APPROVED** [review]

**Pull request merged**